tau2_integration/runners/saga_runner.py

```
@@ -0,0 +1,422 @@
+                tool_calls=self._executed_actions,
+                compensation_actions=compensation_actions,
+                error=str(e),
+            )
+    
+    def _generate_plan(
+        self,
+        llm: Any,
+        task: Tau2TaskDefinition,
+        tools: Dict[str, Any],
+        policy: str,
+        run_config: Optional[Dict[str, Any]] = None,
```

Ruff (0.14.13)

Unused method argument: `run_config`

(ARG002)